Python/AudioBook/Audiobook.py

```diff
@@ -0,0 +1,12 @@
+import pyttsx3
+import PyPDF2
+book=open('software.pdf','rb')
+pdfReader=PyPDF2.PdfFileReader(book)
+pages=pdfReader.numPages

```

Comment:
You can use Dynamic Input as well to take the number of Pages from the User using Argument Parsers.